pkg/account/main.go

```diff
@@ -25,6 +25,10 @@ func Get(jaeger *v1.Jaeger) []*corev1.ServiceAccount {
 func getMain(jaeger *v1.Jaeger) *corev1.ServiceAccount {
 	trueVar := true
 	return &corev1.ServiceAccount{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
Why has this changed?

Reply:
This was just missing. All other types had a correct TypeMeta, but ServiceAccount did not. I guess the k8s API or client adds defaults which is why it was not a problem.

The e2e tests will catch if something else misses this, as long as it is a part of allInOne.